api/client/client.go

```diff
@@ -145,6 +146,26 @@ func (c *Client) ModelDelete(storeID, modelName string, entityIDs ...string) err
 	return err
 }
 
+// CreateInvite retrives the components required to create a Thread/store invite.
+func (c *Client) CreateInvite(storeID string) ([]string, error) {
```

Comment:
Could we call this `GetStoreLink` instead? `CreateInvite` sounds like something is being created to me... where in fact we're just retrieving a pointer to the store.

Reply:
Yup, great idea!